README.rst

```diff
@@ -102,7 +102,7 @@ instead.
 * ``$CWD/tox.ini``
 * ``$CWD/pep8.ini``
 * ``$CWD/setup.cfg``
-* ``$CWD/pyproject.toml`` in section ``[tool.doc8]`` if ``tomli`` is installed
+* ``$CWD/pyproject.toml`` in section ``[tool.doc8]`` if ``tomli`` is installed or when using a Python version greater or equal than 3.11
```

Comment:
Instead of causing different behavior based on presence/absense of another library, we want to keep tomli as a requirement for py<3.11.